Sources/GRPCCodeGen/SourceGenerator.swift

```diff
@@ -28,15 +28,24 @@ public struct SourceGenerator: Sendable {
   public struct Configuration: Sendable {
     /// The access level the generated code will have.
     public var accessLevel: AccessLevel
+    /// Whether imports have explicit access levels.
+    public var accessLevelOnImports: Bool
     /// The indentation of the generated code as the number of spaces.
     public var indentation: Int
     /// Whether or not client code should be generated.
     public var client: Bool
     /// Whether or not server code should be generated.
     public var server: Bool
 
-    public init(accessLevel: AccessLevel, client: Bool, server: Bool, indentation: Int = 4) {
+    public init(
```

Comment:
Not related to this change but this init is missing docs

Reply:
Fixed!